tasks/src/main/java/org/example/fBinaryNotation/FBinaryNotation.java

```diff
@@ -0,0 +1,60 @@
+package org.example.fBinaryNotation;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+/**
+ * https://codeforces.com/problemset/problem/188/F
+ * WARNING: Submit only in Roco language!!! It is not possible to submit code in Java.
+ */
+public class FBinaryNotation {
+    public static void main(String[] args) {
+        FBinaryNotation myMain = new FBinaryNotation();
+
+        FBinaryNotation.FastReader sc = new FastReader();
+        int n = sc.nextInt();
+        myMain.binaryNotation(n);
+    }
+
+    public int binaryNotation(int n) {
+        int[] binaryArr = new int[21];
+        int index = 0;
+        StringBuilder binaryBuilder = new StringBuilder();
+
+        while (n > 0) {
+            binaryArr[index++] = n % 2;
+            n /= 2;
+        }
+
+        for (int i = index - 1; i >= 0; i--) {
+            binaryBuilder.append(binaryArr[i]);
+        }
+        return Integer.parseInt(binaryBuilder.toString());
+    }
+
+    static class FastReader {
```

Comment:
Fajne zastosowanie statycznej klasy

Reply:
Jest to jedna z opcji, aby przechodziły testy na Codeforces. Zauważyłem też, że na tym etapie (zadania, które rozwiązuję) wystarczy zmienić Javę z 17 na 11 lub 1.8 i czas uruchamiania oraz zużycie pamięci znacznie spada.

Reply:
a okej, jezeli tak to spoko niech zostanie